cmd/agent/daemon/daemon.go

```diff
@@ -75,6 +76,9 @@ func NewRunCommand(version string) *cobra.Command {
 		kubeAPIServiceAddr = pflag.String("kube-api-service-addr", "", "Custom kube API service grpc address")
 
 		exportersQueueSize = pflag.Int("exporters-queue-size", 4096, "Exporters queue size")
+
+		redactSensitiveValues         = pflag.Bool("redact-sensitive-values", false, "Redact sensitive values from process exec args")
```

Comment:
I think we don't need this bool flag. If regex string is set we can enable redaction auomatically.

Reply:
but this means we cannot have the default regex.